doc/api/process.markdown

```diff
@@ -218,7 +221,7 @@ Note:
   the terminal mode before exiting with code `128 + signal number`. If one of
   these signals has a listener installed, its default behavior will be removed
   (Node.js will no longer exit).
-- `SIGPIPE` is ignored by default, it can have a listener installed.
+- `SIGPIPE` is ignored by default. It can have a listener installed.
```

Comment:
I don't think it's wrong. Writing to a pipe that can't read results in both `SIGPIPE` signal *and* `EPIPE` error. The signal is ignored while the error is not. At least, that's how I'm seeing it. Correction welcome. And I suspect the docs can be clarified on that. (Feel free to PR something better if you have a good idea.)